### PR TITLE
added initial setup of sulu e-commerce documentation

### PR DESCRIPTION
**Pull request by turbo-ele**
initial content structure (based on sulu-docs):
- book
  - introduction
    - what is sulu e-commerce?
  - getting started
    - installation
- developer guide

### CODE REVIEW

**Comment by christianbader** [comment]
ack

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
 # sulu-ecommerce-docs
 Sulu eCommerce Documentation
+
+## Installation
+To contribute to sulu-documentation you first need to install [sphinx](http://sphinx-doc.org/install.html).
```
Comment (dantleech) [inline README.md]: Add blank line after headings

Comment (dantleech) [inline README.md]: Should be `Sulu documentation`.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+
+```
+    pip install sphinx
+
+    pip install sphinx_rtd_theme
+```
```
Comment (dantleech) [inline README.md]: This is markdown, it should be

````
.. code-block:: bash

    $ pip install sphinx
    $ pip install ...
````

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+
+## Build
+
+```
+    make html 
+```
```
Comment (dantleech) [inline README.md]: Same as above

File: README.md (hunk 2 above)
Comment (dantleech) [inline README.md]: oh .. this *is* markdown :/

**Pull request merged**